src/test/java/uk/gov/pay/products/client/RestClientFactoryTest.java

```diff
@@ -9,36 +14,59 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.core.IsNot.not;
-import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
+@ExtendWith(MockitoExtension.class)
```

Comment:
What's the difference between this and the standard `@RunWith(MockitoJUnitRunner.class)`?

Reply:
I changed RestClientFactoryTest to use JUnit 5 instead of JUnit 4 as mentioned on the commit message. According to JUnit 5 documentation, `@RunWith` no longer exists and is superseded by `@ExtendWith`.

`MockitoExtension` is an extension that initialises mocks and handles strict stubbings. This extension is the JUnit Jupiter (JUnit5) equivalent of our JUnit4 `MockitoJUnitRunner`. Source: https://github.com/mockito/mockito/blob/release/2.x/subprojects/junit-jupiter/src/main/java/org/mockito/junit/jupiter/MockitoExtension.java

Reply:
👍 (I've not used Junit5 yet so this is useful)